### DISCUSSION TITLE
uglyemail.js:10 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'each')

### DISCUSSION THREAD

**Discussion by bendono** [OneClickLab/ugly-email-extension#42] [General]
In the last week I have started getting the below error when I login to a second Gmail session:
  Oops… the system encountered a problem (#2014)
At first it was on just one PC but over a few days it spread to others. This only happened in Chrome while Firefox was fine.
After a bit of investigation I would [this post](https://support.google.com/mail/thread/7717995?hl=en&sjid=10987379785814083416-AP) which helped me identity the issue. After checking the Chrome debug console, I found the following error:
 This document requires 'TrustedHTML' assignment.
```
uglyemail.js:10 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'each')
    at a.observe.initialize_dom_observers (uglyemail.js:10:29862)
    at a.observe.on_dom (uglyemail.js:10:30669)
    at a.observe.on (uglyemail.js:10:26766)
    at uglyemail.js:26:1368
    at Generator.next (<anonymous>)
    at o (uglyemail.js:26:886)
```
Disabling Ugly Email immediately resolved the issue.

Best regards -- Ben

**Comment by sonnyt** [comment]
This is a known bug and being tracked here #38

**Reply by bendono** [reply]
@sonnyt Thanks for the update. Apologies for the repeat. Please feel free to close this.
